### PR TITLE
new RELEASEME file with informations how to deploy this project

### PR DESCRIPTION
**Pull request by muellenborn**

### CODE REVIEW

File: RELEASEME.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+### Sonatype Account
+
+ * If you did not own a sonatype account so far, you have to sign up at [Sonatype](http://central.sonatype.org/pages/ossrh-guide.html)
+ * First create a jira account
+ * Request for publishing permissions in the group "de.kaufhof" via jira ticket
+ * to learn how to use sonatype with your credentials, check [this](http://www.scala-sbt.org/0.13/docs/Using-Sonatype.html#Fourth+-+Adding+credentials)
+
+### SBT-PGP Plugin
+
+ * add `addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8")`to your project's build.sbt, this allows you to sign and publish your artefacts by running publishSigned in sbt
```
Comment (magro) [inline RELEASEME.md]: Shouldn't this go into ~/.sbt...?

Comment (magro) [inline RELEASEME.md]: Just saw the paragraph "GPG SBT-File" below - this should be enough?

File: RELEASEME.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ * btw. builds from this project are made with travis, you can see an overview [here](https://travis-ci.org/Galeria-Kaufhof/ha-jobs) 
+ 
+
+## Snapshot Release
+
+ * Instead of the final release, a snapshot release can be changed and released several times
+ * a SNAPSHOT will NOT be released at maven central but at sonatype, where it can be used from
+ * To build a SNAPSHOT go to your project build.sbt and change the projectVersion by increment the version number and 
+   add -SNAPSHOT behind (e.g. `projectVersion = "1.2.4-SNAPSHOT"`)
+ * there are also version numbers mentioned in the README.md, change them also to the version number you used in the build.sbt
```
Comment (magro) [inline RELEASEME.md]: IMO the Readme should only mention releases, therefore I'd drop this note here.

File: RELEASEME.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+ * the final public release will go out to maven central and, therefore, cannot be released twice with the same version number
+ * to build a regular release just increment the version number in your project build.sbt AND the README.md without adding
+   -SNAPSHOT at the end. If you released a SNAPSHOT with this code stand before do not incremnt the verison number, just remove the
+   -SNAPSHOT ending
+ * start sbt on your project root
+ * publish the changes with `very publishSigned`
+ * confirm the publish with the your gpg password when sbt asks you
+ 
+
+## Sonatype
```
Comment (magro) [inline RELEASEME.md]: The following are steps of the final release, so this heading can be removed or should be h3 at least.

File: RELEASEME.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+   add -SNAPSHOT behind (e.g. `projectVersion = "1.2.4-SNAPSHOT"`)
+ * there are also version numbers mentioned in the README.md, change them also to the version number you used in the build.sbt
+ * start sbt on your project root
+ * publish the changes with `very publishSigned`
+ * confirm the publish with the your gpg password when sbt asks you
+ 
+
+## Final release
+
+ * the final public release will go out to maven central and, therefore, cannot be released twice with the same version number
```
Comment (magro) [inline RELEASEME.md]: Precisely, it's not possible to release the same version after closing the deploy repo. This sentence might lead to a misunderstanding of the process.

File: RELEASEME.md (hunk 1 above)
Comment (muellenborn) [inline RELEASEME.md]: yeah, you're right, i'll delete the SBT-PGP Plugin section...

File: RELEASEME.md (hunk 2 above)
Comment (muellenborn) [inline RELEASEME.md]: done!

File: RELEASEME.md (hunk 3 above)
Comment (muellenborn) [inline RELEASEME.md]: headline Sonatype removed...

File: RELEASEME.md (hunk 4 above)
Comment (muellenborn) [inline RELEASEME.md]: ok, i'll put this information at the point where the repo is closed.

**Pull request merged**